packages/bspwm/common.patch

```diff
@@ -0,0 +1,11 @@
+--- src/common.h	2020-08-02 15:51:32.000000000 +0800
++++ src.mod/common.h	2022-03-21 21:24:04.760854462 +0800
```

Comment:
Use `./src/common.h`, and `./src.mod/common.h` to hopefully make it find the file

Reply:
Helpers.c also same?

Reply:
Yes